aci/resource_aci_l3extlifp.go

```diff
@@ -477,10 +515,24 @@ func resourceAciLogicalInterfaceProfileRead(ctx context.Context, d *schema.Resou
 		relParams := l3extRsLIfPToNetflowMonitorPolData.([]map[string]string)
 		for _, obj := range relParams {
 			relParamList = append(relParamList, map[string]string{
-				"tn_netflow_monitor_pol_name": obj["tnNetflowMonitorPolName"],
-				"flt_type":                    obj["fltType"],
+				"tn_netflow_monitor_pol_dn": obj["tDn"],
+				"flt_type":                  obj["fltType"],
 			})
 		}
+		if relationTol3extRsLIfPToNetflowMonitorPol, ok := d.GetOk("relation_l3ext_rs_l_if_p_to_netflow_monitor_pol"); ok {
+			relationParamListUser := relationTol3extRsLIfPToNetflowMonitorPol.(*schema.Set).List()
+			for _, relationParamUser := range relationParamListUser {
+				paramMapUser := relationParamUser.(map[string]interface{})
+				if paramMapUser["tn_netflow_monitor_pol_dn"] == "" {
+					for _, sub_attributes_map_apic := range relParamList {
+						if sub_attribute_apic_key, ok := sub_attributes_map_apic["tn_netflow_monitor_pol_dn"]; ok {
+							sub_attributes_map_apic["tn_netflow_monitor_pol_name"] = sub_attribute_apic_key
+							delete(sub_attributes_map_apic, "tn_netflow_monitor_pol_dn")
```

Comment:
Does this still work when using tn_netflow_monitor_pol_name? (a warning is displayed but it should still work)

Reply:
Yes. This piece of code here is to achieve idempotent behaviour based on the provided input(tn_netflow_monitor_pol_name vs tn_netflow_monitor_pol_dn). The relationship attribute is of type Set.